jest.setup.ts

```diff
@@ -2,3 +2,4 @@
 // Learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom';
 import '@testing-library/jest-dom/extend-expect';
+import 'jest-canvas-mock';
```

Comment:
Note to reviewer: Necessary for Jest to run a test on pages that contain canvas. [Reference](https://tnodes.medium.com/setting-up-jest-with-react-and-phaser-422b174ec87e#:~:text=That%20null%20is%20actually%20supposed%20to%20be%20the%20canvas%20element.%20Jest%20uses%20jsdom%20as%20a%20psuedo%20DOM%2C%20but%20it%20doesn%E2%80%99t%20include%20the%20canvas%20straight%20away.%20You%20must%20add%20in%20canvas%20support%20so%20jsdom%20can%20find%20it.)